internal/server/errors_test.go

```diff
@@ -41,6 +41,18 @@ func TestSendAPIError(t *testing.T) {
 			err:    fmt.Errorf("hide this: %w", internal.ErrUnauthorized),
 			result: api.Error{Code: http.StatusUnauthorized, Message: "unauthorized"},
 		},
+		{
+			err:    data.ErrAccessKeyExpired,
+			result: api.Error{Code: http.StatusUnauthorized, Message: "unauthorized: " + data.ErrAccessKeyExpired.Error()},
+		},
+		{
+			err:    data.ErrAccessKeyDeadlineExceeded,
+			result: api.Error{Code: http.StatusUnauthorized, Message: "unauthorized: " + data.ErrAccessKeyDeadlineExceeded.Error()},
+		},
+		{
+			err:    fmt.Errorf("hide this: %w", internal.ErrUnauthorized),
+			result: api.Error{Code: http.StatusUnauthorized, Message: "unauthorized"},
+		},
```

Comment:
I think this test case may have been duplicated. Seems like we have it just above the two new ones as well.